vllm/entrypoints/openai/cli_args.py

```diff
@@ -86,6 +86,10 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                         default=None,
                         help="Host name.")
     parser.add_argument("--port", type=int, default=8000, help="Port number.")
+    parser.add_argument("--uds",
+                        type=optional_type(str),
+                        default=None,
+                        help="Unix domain socket name.")
```

Comment:
Some more points to cover here:

- If used, it takes precedence over the options for setting up a TCP or TCP+SSL socket for the API server. Only one or the other will be used, not both.